### PR TITLE
Create preference entry for legacy

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.preferences/plugin.xml [hunk 1]
```diff
@@ -60,6 +61,15 @@
                id="org.eclipse.ui.net.proxy">
          </keywordReference>
       </page>
+      <page
+            category="org.bonitasoft.studio.preferences.pages.BPM"
+            class="org.bonitasoft.studio.preferences.pages.Legacy6xPreferencePage"
+            id="org.bonitasoft.studio.preferences.legacy6x"
+            name="Legacy 6.x">
+         <keywordReference
+               id="org.bonitasoft.studio.preferences.keywordReference1">
```
Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.preferences/plugin.xml]: org.bonitasoft.studio.preferences.keywordReference1 seems no a meaningfull id

File: bundles/plugins/org.bonitasoft.studio.preferences/src/org/bonitasoft/studio/preferences/PreferenceInitializer.java [hunk 2]
```diff
@@ -42,36 +42,74 @@
...
+
+    protected IPreferenceStore getWebBrowserPreferenceStore() {
+        return WebBrowserUIPlugin.getInstance().getPreferenceStore();
+    }
+
+    protected IPreferenceStore getBonitaPreferenceStore() {
+        return BonitaStudioPreferencesPlugin.getDefault().getPreferenceStore();
+    }
+
+    protected void initDefaultWebPreferences(final IPreferenceStore bonitaSstore, final IPreferenceStore webStore) {
```
Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.preferences/src/org/bonitasoft/studio/preferences/PreferenceInitializer.java]: typo bonitaSstore

File: bundles/plugins/org.bonitasoft.studio.preferences/src/org/bonitasoft/studio/preferences/pages/Legacy6xPreferencePage.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+public class Legacy6xPreferencePage extends AbstractBonitaPreferencePage {
+
+    public Legacy6xPreferencePage() {
+        super(GRID);
+        setPreferenceStore(BonitaStudioPreferencesPlugin.getDefault().getPreferenceStore());
+    }
+
+    @Override
+    protected void createFieldEditors() {
+        createTitleBar(Messages.BonitaPreferenceDialog_legacy6x, Pics.getImage(PicsConstants.preferenceLegacy6x), false);
```
Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.preferences/src/org/bonitasoft/studio/preferences/pages/Legacy6xPreferencePage.java]: I'm not a big fan of using Pics.getImage(String).
This image will not be reused elsewhere, it should be defined the closest possible way of where it is used.

Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.preferences/src/org/bonitasoft/studio/preferences/pages/Legacy6xPreferencePage.java]: we can fill a TR for that. The goal of that was also that it is easier for OEM to provide their own icon

Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.preferences/src/org/bonitasoft/studio/preferences/pages/Legacy6xPreferencePage.java]: I'm quite suspicious about the fact that an oem will change anything else but the logo...
But OK, we should talk about that later.

**Pull request merged**